src/pytorch_lightning/utilities/logger.py

```diff
@@ -90,7 +90,7 @@ def _flatten_dict(params: Dict[Any, Any], delimiter: str = "/") -> Dict[str, Any
 
     def _dict_generator(
         input_dict: Any, prefixes: List[Optional[str]] = None
-    ) -> Generator[Any, Optional[List[str]], List[Any]]:
+    ) -> Generator[Any, Optional[List[str]], List[Any]]:  # FIXME
```

Comment:
FIXME: Not sure about this one yet...

Reply:
Maybe ignore it? 😃

Reply:
I instead simplified the implementation so typing is not ambiguous